File: Basic Coding Problems/02 If - Elses/023 Write a program to check whether a number is divisible by 5 and 11 or not/C/divisible_by_5_and_11.c
#include <stdio.h>

int main() {
    int number;

    printf("Enter a number: ");
    scanf("%d", &number);

    if (number % 5 == 0 && number % 11 == 0) {
        printf("%d is divisible by 5 and 11.\n", number);
    } else {
        printf("%d is not divisible by 5 and 11.\n", number);
    }

    return 0;
}

File: Basic Coding Problems/02 If - Elses/023 Write a program to check whether a number is divisible by 5 and 11 or not/Java/divisible_by_5_and_11.java
import java.util.Scanner;

public class divisible_by_5_and_11 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter a number: ");
        int number = scanner.nextInt();

        if (number % 5 == 0 && number % 11 == 0) {
            System.out.println(number + " is divisible by 5 and 11.");
        } else {
            System.out.println(number + " is not divisible by 5 and 11.");
        }
    }
}